.github/actions/breaking-pr-check/index.js

```diff
@@ -0,0 +1,66 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+function raiseError(message) {
+  throw new Error(message);
+}
+
+async function getPullRequest() {
+  const client = github.getOctokit(process.env.GITHUB_TOKEN);
+
+  const pr = github.context.payload.pull_request;
+  if (!pr) {
+    throw new Error(
+      "This action can only be invoked in `pull_request_target` or `pull_request` events. Otherwise the pull request can't be inferred.",
+    );
+  }
+
+  const owner = pr.base.user.login;
+  const repo = pr.base.repo.name;
+
+  const { data } = await client.rest.pulls.get({
+    owner,
+    repo,
+    pull_number: pr.number,
+  });
+
+  return data;
+}
+
+function checkTitle(title) {
+  if (/^[a-z]+(\([a-z]+\))?!: /.test(title)) {
```

Comment:
```suggestion
  if (/^[a-z]+(\([a-z-]+\))?!: /.test(title)) {
```

Without the dash, this regexp won't catch `feat(eslint-plugin)!: ...`

Reply:
Thanks! I fixed it [cde7ec4](https://github.com/typescript-eslint/typescript-eslint/pull/9050/commits/cde7ec408fdd2b7a716c1b54e8062ddcefc18219)